### DISCUSSION TITLE
Lack of therapy (insulin, carbohydrates) in the follower (xdrip sync)

### DISCUSSION THREAD

**Discussion by Inkalika** [NightscoutFoundation/xDrip#3223] [Bug reports]
Previously, the version dated 09.2023 was installed on the master and follower. Libre2, 5 minute values. In the follower xdrip sync is selected as the source.
At night on the follower the Internet turns off and turns on in the morning. After being included in the follower, all therapy (carbohydrates, insulin) for the night and all sugar values are downloaded.

I updated the version to 2023.11.29 on master and follower and enabled the transfer of 1 minute values.
1) The follower does not receive every value, but one value every 3-4 minutes 
(Each 1-minute value is transmitted to the Nightscout. But the Nightscout also does not pump up therapy data overnight. But this was also the case in previous versions, so I'm already used to it.)
2) After turning off the Internet at night and turning it on in the morning, therapy (insulin, carbohydrates) is not downloaded overnight, and sugar values are only downloaded for the last half of the night.
I returned the version on the follower to the previous one, but it didn’t help. I assume that this situation was caused by the inclusion of 1-minute values, or something else was corrected, since previously therapy was pumped into the follower after turning on the Internet.

**Comment by Navid200** [comment]
If you install the same version that you were previously using on both master and follower again, will this problem go away?

You can easily install that version over.  You don't need to uninstall first.

**Comment by Inkalika** [comment]
I installed the old version, but left the transmission of 1-minute sugar values. After turning off the Internet on the follower for 6 hours and restoring the Internet, therapy did not recover during the absence of the Internet, and sugar values were displayed in the last 5 hours.

I will try to turn off the transmission of 1-minute values and see if the therapy will be downloaded for the entire period of absence of the Internet.

And how should the хdrip program behave according to the developer's idea after restoring the Internet on the follower (xdrip sunc)? Should all sugar and therapy values be displayed during the absence of the Internet? Or should they be displayed only for a certain period?

And is it normal that not every 1-minute value is transmitted to the follower (xdrip sunc), but only one value every 3-4 minutes?
